### ISSUE TITLE
Can't have a module called "React" and run `dune utop`

### ISSUE THREAD

**Issue by davesnx** [ocaml/dune#10656]
## Expected Behavior

Load utop with a file called React

## Actual Behavior

→ ~/C/g/m/server-reason-react on main ✓ dune utop
File "_none_", line 1:
Error: Files packages/react/src/react.cma(React)
       and foo/server-reason-react/_opam/lib/react/react.cma(React)
       both define a module named React

## Reproduction

git clone https://github.com/ml-in-barcelona/server-reason-react
make init
dune utop

Related issue: https://github.com/ml-in-barcelona/server-reason-react/issues/38

**Comment by nojb** [comment]
I don't think there is much Dune can do here; `utop` already links to `React`, so you cannot link it again to it...

**Comment by davesnx** [comment]
What about vendor React and Lwt inside dune and prefix them?

**Comment by nojb** [comment]
> What about vendor React and Lwt inside dune and prefix them?

This strikes me as something that should rather be suggested to the utop maintainers, do you agree?

**Comment by davesnx** [comment]
Yes, that falls into utop. I will reopen the issue there: https://github.com/ocaml-community/utop/issues/463
